CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 CHANGELOG
 =========
 
+dev-develop
+-----------
+
+### Feature
```

Comment:
Can you maybe convert the entire file to use the markdown syntax, as I did in Husky? It's somehow embarrassing that all our repos have different styleguides on this :confused: